skyjo.py

```diff
@@ -74,8 +75,28 @@
 Deck: TypeAlias = np.ndarray[tuple[int], np.uint8]
 """Cards remaining in the deck in lieu of a random seed."""
 
-Skyjo: TypeAlias = tuple[Game, Table, Deck, int, int, int | None]
-"""A lightweight representation of a Skyjo game."""
+Skyjo: TypeAlias = tuple[Game, Table, Deck, int, int, int | None, int | None]
+"""A lightweight representation of a Skyjo game.
+
+Values:
+    game: Game
+        non-board game state values like discard pile, scores, etc.
+    table: Table
+        board of cards for each player
+    deck: Deck
+        remaining card counts does not include top card
+    players: int
+        number of players
+    current_player: int
+        current acting player index
+    card: int | None 
+        for use in random outcome
+    countdown: int 
+        number of cards remaining in the deck
+"""
+
+SkyjoAction: TypeAlias = int
+"""Integer representing an action in the Skyjo game."""
```

Comment:
Nothing to do with this particular PR, but consider doing this for `int -> Card` and adding documentation about how it's an index not a value (I wonder if `NewType` would be good here)